cmd/kubemark/hollow-node.go

```diff
@@ -70,13 +72,39 @@ func makeTempDirOrDie(prefix string, baseDir string) string {
 	return tempDir
 }
 
+func createClientFromFile(path string) (*client.Client, error) {
+	c, err := clientcmd.LoadFromFile(path)
+	if err != nil {
+		return nil, fmt.Errorf("Error while loading kubeconfig from file %v: %v", path, err)
```

Comment:
nit: error strings should start with a lower case letter. 

https://github.com/golang/go/wiki/CodeReviewComments#error-strings